Lib/test/test_peepholer.py

```diff
@@ -409,21 +409,6 @@ def f(cond1, cond2):
         self.assertLessEqual(len(returns), 6)
         self.check_lnotab(f)
 
-    def test_elim_jump_after_return2(self):
```

Comment:
I've removed this test, as doesn't test what it claims to test. There is no dead code in
```
def f(cond1, cond2):
    while 1:
        if cond1: return 4
```